src/main/java/org/opentripplanner/routing/impl/StreetVertexIndexServiceImpl.java

```diff
@@ -69,33 +74,15 @@
 
     static final Logger LOG = LoggerFactory.getLogger(StreetVertexIndexServiceImpl.class);
 
-    private SimpleStreetSplitter simpleStreetSplitter;
+    private StreetSplitter streetSplitter;
 
     public StreetVertexIndexServiceImpl(Graph graph) {
-        this(graph, true);
-    }
-
-    public StreetVertexIndexServiceImpl(Graph graph, boolean hashGrid) {
```

Comment:
The only place this method signature was called from was the above constructor. Therefore, all code paths where `hashGrid` is false are impossible to reach. Therefore, I removed those code paths.

Reply:
I would take this refactor a step farther and just get rid of StreetVertexIndexServiceImpl, DefaultStreetVertexIndexFactory, and StreetVertexIndexFactory.  I just added an issue for this at #2770